docs/user-guide-for-quickstart.md

```diff
@@ -0,0 +1,133 @@
+---
+id: user-guide-for-quickstart
+title: Quickstart Tour
+sidebar_label: Tour
+---
+
+StreamPipes is a framework that enables domain experts to model and execute stream processing pipelines in a big data infrastructure.
+The graphical user interface of StreamPipes is a web application that provides an easy to use solution for domain experts.
+
+Quickstart Mode, ideal for first-time users, allows users to interact with pre-set pipelines, dashboards, and data views to experience StreamPipes' functionalities for IIoT. 
+This page provides an overview of StreamPipes' features in Quickstart Mode, guiding users on how to utilize them effectively.
+
```

Comment:
```suggestion
:::info
Do you have StreamPipes already running in the quickstart mode? If not simply run the command `docker-compose -f docker-compose.quickstart.yml up -d` from the directory `installer/compose`.
:::
```

Reply:
Hi, @bossenti ! I have added the part as the suggestion, The `info` format looks really cool! Thanks a lot!